src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -0,0 +1,389 @@
+  }
+
+  static defaultProps = {
+    srsName: 'EPSG:3857',
+    outputFormat: 'application/json',
+    minChars: 3,
+    additionalFetchOptions: {},
+    /**
+     * Create an AutoComplete.Option from the given data.
+     *
+     * @param {Object} feature The feature as returned by the server.
+     * @return {AutoComplete.Option} The AutoComplete.Option that will be
+     * rendered for each feature.
+     */
+    renderOption: feature => {
+      return (
+        <Option key={feature.id}>
+          {feature.properties.name ? feature.properties.name : feature.id}
+        </Option>
+      );
+    },
+    /**
+     * The default onSelect method if no onSelect prop is given. It zooms to the
+     * selected item.
+     *
+     * @param {object} feature The selected feature as returned by the server.
+     * @param {ol.map} olMap The openlayers map that was passed via prop.
+     */
+    onSelect: (feature, olMap) => {
+      if (feature) {
+        const olView = olMap.getView();
+        const geoJsonFormat = new OlFormatGeoJSON();
+        const olFeature = geoJsonFormat.readFeature(feature);
+        const geometry = olFeature.getGeometry();
+
+        if (geometry) {
+          olView.fit(geometry, {
+            duration: 500
+          });
+        }
+      }
+    },
+    style: {
+      width: 200
+    }
+  }
+
+  /**
+   * Create the WfsSearch.
+   *
+   * @param {Object} props The initial props.
+   * @constructs WfsSearch
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTerm: '',
+      data: []
+    };
+    this.onUpdateInput = this.onUpdateInput.bind(this);
+    this.onMenuItemSelected = this.onMenuItemSelected.bind(this);
+  }
+
+  /**
+   * Called if the input of the AutoComplete is being updated. It sets the
+   * current inputValue as searchTerm and starts a search if the inputValue has
+   * a length of at least `this.props.minChars` (default 3).
+   *
+   * @param {String|undefined} value The inputValue. Undefined if clear btn
+   *                                      is pressed.
+   */
+  onUpdateInput(value) {
+    this.setState({
+      data: []
+    });
+
+    if (value) {
+      this.setState({
+        searchTerm: value || ''
+      }, () => {
+        if (this.state.searchTerm.length >= this.props.minChars) {
+          this.doSearch();
+        }
+      });
+    }
+  }
+
+  /**
+   * Perform the search.
+   * @private
+   */
+  doSearch() {
+    const {
+      featureNS,
+      featurePrefix,
+      featureTypes,
+      geometryName,
+      maxFeatures,
+      outputFormat,
+      propertyNames,
+      srsName,
+      baseUrl,
+      additionalFetchOptions
+    } = this.props;
+
+    const options = {
+      featureNS,
+      featurePrefix,
+      featureTypes,
+      geometryName,
+      maxFeatures,
+      outputFormat,
+      propertyNames,
+      srsName,
+      filter: this.createFilter()
+    };
+
+    const wfsFormat = new OlFormatWFS();
```

Comment:
By default `OlFormatWFS` supports WFS version 1.1. Please make the used WFS version configurable (see e.g. [here](https://openlayers.org/en/latest/apidoc/ol.format.WFS.html)).

WFS 2.0.x is not supported, or am I wrong?

Reply:
`ol.format.WFS` does not support WFS 2.0.0, is it really necessary to support WFS 1.0.0?

Reply:
no need for 1.0.0 IMO. 2.0.0 would be nice, but is not a must have.

Reply:
Since WFS 1.0.0 will be generated simply by using another GML format in `OlFormatWFS` I would make this configurable

Reply:
I :womans_hat: this done in 638c12d